apps/arweave/src/ar_util.erl

```diff
@@ -381,3 +381,23 @@ assert_file_exists_and_readable(FilePath) ->
 			io:format("~nThe filepath ~p doesn't exist or isn't readable.~n~n", [FilePath]),
 			erlang:halt(1)
 	end.
+
+%% @doc Clamp a value to a lower bound.
+%% If the value is less than the lower bound, return the lower bound.
+clamp_lower(Value, LowerBound) when Value < LowerBound ->
+	LowerBound;
+clamp_lower(Value, _LowerBound) ->
+	Value.
+
+%% @doc Clamp a value to an upper bound.
+%% If the value is greater than the upper bound, return the upper bound.
+clamp_upper(Value, UpperBound) when Value > UpperBound ->
+	UpperBound;
+clamp_upper(Value, _UpperBound) ->
+	Value.
+
+%% @doc Clamp a value to a lower and upper bound.
+%% If the value is less than the lower bound, return the lower bound.
+%% If the value is greater than the upper bound, return the upper bound.
+clamp(Value, LowerBound, UpperBound) ->
```

Comment:
FYI: `ar_util:between/3` also does this: https://github.com/ArweaveTeam/arweave/pull/660/files#diff-1f26621f4ea9ef47e071afce0f4ce9fc3e6bacc76089ac9037c00e0c07a5ce9eR46

Can you add a comment to `ar_util:between` that people should start using ar_util:clamp instead?

Reply:
Ah, didn't see the existing one, okay.